doc/source/conf.py

```diff
@@ -56,7 +56,7 @@
 
 # General information about the project.
 project = u'Trident'
-copyright = u'2015-2017, Trident Project'
+copyright = u'2015-2018, Trident Project'
```

Comment:
Recommend changing this to `Trident Project. All rights reserved.` so we don't have to keep updating the date.  Projects have been doing this lately.